packages/utils/src/address.ts

```diff
@@ -19,5 +25,7 @@ export const publicKeyToAddress = (publicKey: string) => {
 export const privateKeyToPublicKey = (privateKey: string) => {
   return '0x' + privateToPublic(Buffer.from(privateKey.slice(2), 'hex')).toString('hex')
 }
+export const isValidPrivateKey = (privateKey: string) =>
+  isValidPrivate(Buffer.from(privateKey.slice(2), 'hex'))
```

Comment:
Would be nice if this and the functions above it only did the slice after confirming the leading chars are '0x'
So happens @yorhodes added some util methods in this file to help with just that:
https://github.com/celo-org/celo-monorepo/pull/1570#discussion_r346573047

Reply:
I added a check that the privateKey starts with `0x` as `isValidPrivate` will then do the rest